homeassistant/components/wallbox/__init__.py

```diff
@@ -37,6 +39,37 @@
 PLATFORMS = [Platform.SENSOR, Platform.NUMBER, Platform.LOCK]
 UPDATE_INTERVAL = 30
 
+CHARGER_STATUS: dict[int, str] = {
```

Comment:
What's the source of this? Please add a link or explanation in a comment.
Please also sort the dict.

Reply:
It is based on the source code of the Wallbox Portal, I placed a comment in the code mentioning this. Also sorted the dict.

Reply:
Can we add a link to the source code?

Reply:
I included a url in the comment